src/libraries/System.Memory/ref/System.Memory.cs

```diff
@@ -732,4 +732,39 @@ namespace System.Text
         public System.Text.SpanRuneEnumerator GetEnumerator() { throw null; }
         public bool MoveNext() { throw null; }
     }
+    public readonly struct RunePosition : System.IEquatable<System.Text.RunePosition>
+    {
+        private readonly int _dummyPrimitive;
+        public static System.Text.RunePosition.Utf16Enumerator EnumerateUtf16(System.ReadOnlySpan<char> span) { throw null; }
+        public static System.Text.RunePosition.Utf8Enumerator EnumerateUtf8(System.ReadOnlySpan<byte> span) { throw null; }
+        public System.Text.Rune Rune { get { throw null; } }
+        public int StartIndex { get { throw null; } }
+        public int Length { get { throw null; } }
+        public bool WasReplaced { get { throw null; } }
+        public RunePosition(Rune rune, int startIndex, int length, bool wasReplaced) { }
```

Comment:
>public RunePosition(Rune rune, int startIndex, int length, bool wasReplaced) { } [](http://example.com/codeflow?start=8&length=80)

Looks the original proposal has no public constructors. If need to add this that need to be reviewed and approved again.